gbeaudry321/ser401-NASA-psyche-mission
# Us375 ip collision

implemented info point appears when player is near info points for psyche, stallite, and stereo images

To test and expected results:
Play the experience
The Psyche mission IP should already be visible at the back of the ship. You should be able to click through the tabs. If you move around the ship, it will remain in position at the back of the ship. If you move far enough away from the ship, it will disappear, then reappear if you move towards it again.
Teleport to the satellite and stereo image zones, approach the Psyche model, a widget should appear and rotate with the character if you move around, unless you get too far away from the psyche model. you should be able to click through the tabs
